resources/views/waitingrooms/partials/_form_waitingrooms.blade.php

```diff
@@ -2,7 +2,7 @@
 
 <div class="form-item -padded">
     {{ Form::label('contest_id', 'Contest ID:', ['class' => 'field-label']) }}
-    {{ Form::text('contest_id', NULL, ['class' => 'text-field']) }}
+    {{ Form::text('contest_id', NULL, ['class' => 'text-field', 'readonly' => 'readonly']) }}
```

Comment:
But yeah, not a dumb question 😉  Totally not obvious off the bat. Not sure if I pass an empty string to the value if it would make it `readonly = ""` or just output `readonly`, but wasn't too concerned to try.